data/templates/accel-ppp/ppp-options.j2

```diff
@@ -0,0 +1,35 @@
+#ppp options
+[ppp]
+verbose=1
+check-ip=1
+ccp={{ "0" if ppp_options.ccp_disable is vyos_defined else "1" }}
+unit-preallocate={{ "1" if authentication.radius.preallocate_vif is vyos_defined else "0" }}
+{% if ppp_options.min_mtu is vyos_defined %}
+min-mtu={{ ppp_options.min_mtu }}
+{% endif %}
+{% if ppp_options.mru is vyos_defined %}
+mru={{ ppp_options.mru }}
+{% endif %}
+mppe={{ ppp_options.mppe }}
+lcp-echo-interval={{ ppp_options.lcp_echo_interval }}
+lcp-echo-timeout={{ ppp_options.lcp_echo_timeout }}
+lcp-echo-failure={{ ppp_options.lcp_echo_failure }}
+{% if ppp_options.ipv4 is vyos_defined %}
+ipv4={{ ppp_options.ipv4 }}
+{% endif %}
+{# IPv6 #}
+{% if ppp_options.ipv6 is vyos_defined %}
+ipv6={{ ppp_options.ipv6 }}
+{%     if ppp_options.ipv6_intf_id is vyos_defined %}
```

Comment:
What will be in this case? Should we check values of `ppp-options ipv6`?
```
set service pppoe-server ppp-options ipv6 deny
set service pppoe-server ppp-options ipv6-intf-id random
```
Another question is whether we probably need to rename `ipv6-intf-id` to `ipv6-interface-id`.

Also, what about moving all related IPv6 options to `ipv6` node?
i.e 
```
set service pppoe-server ppp-options ipv6 negotiate deny|allow|prefer|require
set service pppoe-server ppp-options ipv6 interface-id random
```

Reply:
>we probably need to rename ipv6-intf-id to ipv6-interface-id

Agree.

>Should we check values of ppp-options ipv6?

A strong argument for keeping the CLI as it is now is that there are no IPv4-specific PPP options.
Even in https://docs.accel-ppp.org/configuration/ppp.html I don't see anything that would go under the `ipv4` node. An empty `ipv6` non-leaf node may look confusing, too — is it significant or an empty container?
I'd keep that CLI as is, and we can get back to this if any IPv4-specific options ever appear.

Reply:
done. intf has changed to interface